src/app/components/pages/info-page/info-page.component.html

```diff
@@ -1,67 +1,298 @@
     </p>
 
+    <h3 class="info-page__title">
+      <b><u>Informations importantes à lire concernant Volontaria</u></b>
+    </h3>
 
-    <h4 class="info-page__title">
-      Les missions des bénévoles sont variées. En voici un aperçu :
-    </h4>
     <div class="info-page__content">
+      <h4><u>Inscription</u></h4>
       <p>
-        <b>PRÉ-EMBALLAGE</b> : emballer différents produits,
-        étiqueter des sacs, assembler des commandes, faire
-        la vaisselle…
+        Pour t'inscrire à une plage de bénévolat :
       </p>
+        <ul>
+          <li>Clique sur <i>M’inscrire à une plage horaire</i>, selon la période d’emballage de ton choix.</li>
+          <li>Clique sur <i>Filtrer par cellule</i> et choisis la ville de ton choix.</li>
+          <li>Choisis la ou les plages horaires qui te conviennent et clique sur <i>M’inscrire</i>.</li>
+          <li>Révise les informations concernant la ou les plages horaires que tu as choisies et clique sur <i>M’inscrire comme bénévole</i>.</li>
+        </ul>
+      <p>
+        Attention : Aucune autre confirmation ne te sera envoyée!
+        Dès que tu es inscrit(e), ta participation est confirmée.
+        Tu peux en tous temps aller voir ton horaire dans l’onglet <i>Mon horaire</i>.
+        <br/>Tu peux nous informer par courriel si tu as des problèmes de santé, des restrictions ou des allergies et nous nous adapterons à tes besoins!
+      </p>
+    </div>
+
+    <div class="info-page__content">
+      <h4><u>Désinscription</u></h4>
       <p>
-        <b>EMBALLAGE</b> : peser les aliments des participants
-        en étant attitré à une station.
+        Il est possible de te désinscrire toi-même d'une plage horaire.
+        Il te suffit d'aller dans l'onglet <i>Mon horaire</i> et de cliquer sur le bouton rouge <i>Me désinscrire</i>!
       </p>
     </div>
 
-    <h4 class="info-page__title">
-      Inscription
-    </h4>
     <div class="info-page__content">
+      <h4><u>Le bénévolat c’est du sérieux!</u> :)</h4>
       <p>
-        Pour t'inscrire à une activité, clique sur le bouton à cet
-        effet dans la colonne de droite. Tu seras redirigé vers
-        une page de confirmation.
+        Comme le succès d’une journée d’emballage dépend du nombre suffisant de bénévoles, nous comptons sur ta présence.
+        Cependant, si un changement t’empêche de te présenter, tu peux te désinscrire toi-même.
+        Si c'est possible pour toi de le faire deux ou trois jours à l'avance, c'est très apprécié!
+        <br/><br/>
+        Sinon, désinscris-toi et appelle ou écris au responsable de ta ville de bénévolat afin que nous trouvions un(e) autre bénévole pour combler ton poste dans l'équipe.
+        Merci d'inclure dans ton message ton nom et la plage horaire concernée.
       </p>
+    </div>
+
+    <div class="container">
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
```

Comment:
We don't use Bootstrap or other pulg and play CSS in this project, you need to use a custom BEM SASS.